### PR TITLE
code style fixed with Styler

### PR DESCRIPTION
**Pull request by pouryany**

### CODE REVIEW

File: R/04-miRNAPrioritization.R [hunk 1]
```diff
@@ -30,175 +30,185 @@
...
         # jack.knife.data.filename  <- paste0(prefix, m, "_", samp.rate, "_samples_jack_knifed.RDS")
         # jack.knife.data.file <- paste0(sampling.data.dir, '/', jack.knife.data.filename)
-        # 
-        # jack.knife.data <- sampling.data.base(enrich.null, m.selector, samp.rate, fn, n_paths, 
+        #
+        # jack.knife.data <- sampling.data.base(enrich.null, m.selector, samp.rate, fn, n_paths,
         #                                       sampling.data.file=jack.knife.data.file, jack.knife=TRUE, save.sampling = save.jack.knife)
-        # 
-        
+        #
```
Comment (hackdna) [inline R/04-miRNAPrioritization.R:160]: Remove unused code

File: R/04-miRNAPrioritization.R [hunk 2]
```diff
@@ -30,175 +30,185 @@
...
+          save.sampling = save.sampling,
+          num.cores = 8
+        )
+        m.selector <- jackKnifeBase(
+          selector = m.selector,
+          pathways = pathways,
+          enrich.null = enrich.null,
+          fn = fn,
+          jack.knife.data =
+            sampling.data[[paste0("SampSize_", 100)]],
```
Comment (hackdna) [inline R/04-miRNAPrioritization.R:178]: This can fit on one line now?

File: R/04-miRNAPrioritization.R [hunk 3]
```diff
@@ -30,175 +30,185 @@
...
-                                      !!met2 := stats::p.adjust(p = !!rlang::sym(met)
-                                                         ,method = "fdr")
-                                      )
+      met2 <- paste0(m, "_fdr")
+      selector <- selector %>% dplyr::mutate(
+        .,
+        !!met2 := stats::p.adjust(
+          p = !!rlang::sym(met),
+          method = "fdr"
+        )
```
Comment (hackdna) [inline R/04-miRNAPrioritization.R:200]: This statement can fit on one line now?

**Review by hackdna — CHANGES_REQUESTED** [review]

File: R/04-miRNAPrioritization.R (hunk 2 above)
Comment (pouryany) [reply R/04-miRNAPrioritization.R:178]: The styler package does some magic indeed!

Comment (hackdna) [reply R/04-miRNAPrioritization.R:180]: Yes but it can't fix everything so it still needs some help.

File: R/04-miRNAPrioritization.R [hunk 4]
```diff
@@ -30,175 +30,187 @@
...
+      if (run.jack.knife == T) {
+        # jack.knife.data.filename  <- paste0(prefix, m,
+        #    "_", samp.rate, "_samples_jack_knifed.RDS")
+        # jack.knife.data.file <- paste0(sampling.data.dir, '/',
+        #     jack.knife.data.filename)
+        #
+        # jack.knife.data <- 
+         #   sampling.data.base(enrich.null, m.selector, samp.rate, fn, n_paths,
+        #    sampling.data.file=jack.knife.data.file, jack.knife=TRUE,
+        #    save.sampling = save.jack.knife)
```
Comment (hackdna) [inline R/04-miRNAPrioritization.R:163]: Unused code should be removed.

**Review by hackdna — CHANGES_REQUESTED** [review]

**Review by hackdna — CHANGES_REQUESTED** [review]
I've requested three changes, could you address them?

**Comment by pouryany** [comment]
> I've requested three changes, could you address them?

Done.

**Review by hackdna — CHANGES_REQUESTED** [review]
Thanks, and the unused code in `04-miRNAPrioritization.R`?

**Review by hackdna — APPROVED** [review]

**Pull request merged**